class/Form/ItemForm.php

```diff
@@ -359,6 +359,7 @@ public function createElements($obj)
             }
 
             $dateExpireYesNo     = new \XoopsFormRadioYN('', 'use_expire_yn', $dateexpire_opt);
+            $dateexpire = \strtotime(\formatTimestamp($dateexpire)); //set to user timezone
```

Comment:
Based on the recent discussion: https://xoops.org/modules/newbb/viewtopic.php?start=0&topic_id=78783, we have to get away from using "stringtotime()"
See: https://docs.google.com/document/d/1FEoyDfI96XZn-Ib0Ywigi3nYyj6wkbBagFgbtul0qeA/
Let me know what you think...

Reply:
thanks. I changed this with my PR #137